Bolas update: Traitor items edition

PKPenguin321 opened the pull request:
Adds reinforced bolas, they do a Weaken(1) in addition to normal bola functions and take twice as long to break out of when compared to normal ones. Their sprite has a different color.

Also added a message that displays to everybody when someone is ensnared by a bola.

Moves the throwing star bundle to uplink_kits.dm from weaponry.dm and makes it use a syndicate box.

Tweaked the Throwing Star box to be the Throwing Weapon box, updated uplink listing name and description accordingly. It now contains everything it used to in addition to two reinforced bolas.

Lowered throwing weapon bundle cost to 5 from 6 (it honestly should be lowered even further but I don't want to create controversy). At 6 points, it costed 1 less TC than an energy sword, while being significantly less useful.

Sprite is straight up taken from Paradise. Hopefully this is okay with @Fox-McCloud . I would have asked before PRing, but I don't really know any good way to get into contact.
![bola](https://cloud.githubusercontent.com/assets/8345184/13661990/07fdf478-e64c-11e5-8ac3-97e7efa592c3.png)

:cl: PKPenguin321
rscadd: Added reinforced bolas. They do a very short stun in addition to normal functions, and take twice as long to break out of when compared to regular bolas.
tweak: The traitor's Throwing Star Box has been made into the Throwing Weapons Box. It now contains two reinforced bolas in addition to its old contents. It also costs 5 TCs, down from 6.
/:cl:

PKPenguin321 committed:
adds reinforced bolas, tweaks uplink throwing star bundle

adds reinforced bolas, they do a Weaken(1) and take twice as long to break out of when compared to normal ones. their sprite has a different color
moves throwing star bundle to uplink_kits.dm from weaponry.dm and makes it use a syndicate box
tweaked the throwing star bundle to be the throwing weapon bundle, updated uplink listing and description accordingly
lowered throwing weapon bundle cost to 5 from 6

KazeEspada commented:
Instead of reinforced can they be energy bolas.

KazeEspada commented on code/game/objects/items/weapons/handcuffs.dm:
```diff
@@ -287,14 +287,25 @@
+	desc = "A strong bola, made with a long steel chain. It looks heavy, enough so that it could trip somebody."
+	icon_state = "bola_r"
+	breakouttime = 70
+	stun = 1
```
should be named weaken, and you could use it to set the weaken time as well.

PKPenguin321 replied:
does Weaken(0) do nothing? if so I can just make the var control the exact power of the weaken

KorPhaeron commented:
Make them use the same sprite because meta

Fox-McCloud commented:
>Sprite is straight up taken from Paradise. Hopefully this is okay with @Fox-McCloud . I would have asked before PRing, but I don't really know any good way to get into contact.

It's fine; we're AGPLv3--all I ask is that anything you port from us, you try to mention us, if possible--It really doesn't matter to me, to be honest---we utilize so much from you guys; I'm just happy to see some of our own stuff used elsewhere.

Xhuis commented:
Throwing stars are insanely useful with a pneumatic cannon. Bolas could be even better. Load a pneumatic cannon with throwing stars and a reinforced bola - it's a guaranteed huge amount of damage, embeds, and a quick stun AND legcuff so they can't run away while you bludgeon them.

Supermichael777 commented:
can bolas embed?

RemieRichards commented on code/modules/uplink/uplink_item.dm:
```diff
@@ -513,12 +513,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
+/datum/uplink_item/stealthy_weapons/throwingweapons
+	name = "Box of Throwing Weapons"
+	desc = "A box of shurikens and reinforced bolas from ancient Earth martial arts. They are highly effective \
+			 throwing weapons. The bolas can knock a target down and the shirukens will embed into limbs."
```
shuriken

Xhuis commented:
@Supermichael777 Unless bolas have an embed chance anyway, no.

PKPenguin321 commented:
ughhh why does metastation have a box of throwing stars on the map

PKPenguin321 committed:
maybe fixes metastation

Lumipharon commented:
probs random chance to spawn in the armoury locker.

Also why would you load bolas into a pneumatic when you can insta crit anyone just by loading it full of glass and shit?

And on another note, throwing stars are pretty gay purely because they're sneaky peaky throwing weapons, yet they create this HUGE TEXT saying who's throwing them, removing any chance of stealth (unlike suppressed guns which have no message).

phil235 commented:
>can bolas embed? (whith pnumatic cannon?)

If that's the case we have a bug, the throw_impact() code won't handle this correctly. :disappointed:

PKPenguin321 commented:
pretty sure bolas can't embed

PKPenguin321 committed:
changes var for bola stunning
now instead of a TRUE/FALSE toggle, it's the actual amount of Weaken() applied. default is 0 (no stun)
also fixes small typo in uplink_item.dm

PKPenguin321 commented:
Ready to merge once 24 mark is hit, assuming travis passes
